### ISSUE TITLE
Buttons should not be hidden in certain states; need another pattern to offer better accessibility

### ISSUE THREAD

**Issue by megoth** [SolidOS/solid-panes#53]
The pattern of hiding/showing buttons according to state is problematic in terms of accessibility. It might also not offer the best UX for users. I think toggling availability by changing color and using other cues are a better approach (and mark it semantically, using the `disabled`-attribute).

Some examples of this pattern:
- When offering controls for a resource: Is hidden until you hover with a mouse
- Save-button for sourcePane: I hidden until there are changes to be saved

**Comment by timbl** [comment]
Good point.  Though I think that hiding the optional pane icons at each level made a huge improvement in legibility.   With them in, it looks very geeky and off putting to a normal user, and quite distracting even to a power user.   Maybe some without hide them behind a  "..."  'more stuff' logo.  

Agree about disabled start buttons.   Something like making them 80% transparent would be better.
The disabled flag on the control may not be what we need either ...
